.github/workflows/goreleaser.yml

```diff
@@ -171,4 +171,4 @@ jobs:
       - name: Run installer release workfow
```

Comment:
We shouldn't run this if 'skip_publish' is set.

See https://github.com/mondoohq/cnspec/blob/2202d3cd36d6235527b3c24b69911ba5d6a1cba1/.github/workflows/goreleaser.yml#L142